IdentityCore/src/cache/token/MSIDLegacyTokenCacheItem.m

```diff
@@ -159,6 +160,7 @@ - (void)encodeWithCoder:(NSCoder *)coder
     [coder encodeObject:self.clientId forKey:@"clientId"];
     [coder encodeObject:self.target forKey:@"resource"];
     [coder encodeObject:self.expiresOn forKey:@"expiresOn"];
+    [coder encodeObject:self.extendedExpiresOn forKey:@"extendedExpiresOn"];
```

Comment:
This actually needs to go to the additionalServer dictionary to comply with the old schema.

Reply:
fixed